includes/polling/functions.inc.php

```diff
@@ -243,12 +243,16 @@ function poll_device($device, $force_module = false)
     unset($array);
 
     // Start counting device poll time
-    echo 'Hostname: ' . $device['hostname'] . PHP_EOL;
-    echo 'Device ID: ' . $device['device_id'] . PHP_EOL;
-    echo 'OS: ' . $device['os'];
+    echo 'Hostname:    ' . $device['hostname'] . PHP_EOL;
+    echo 'Device ID:   ' . $device['device_id'] . PHP_EOL;
+    echo 'OS:          ' . $device['os'] . PHP_EOL;
     $ip = dnslookup($device);
 
-    $db_ip = isset($ip) ? inet_pton($ip) : null;
+    $db_ip = null;
+    if (isset($ip) && !empty($ip)) {
```

Comment:
isset case is covered by !empty...
This is an unrelated fix (but a welcome one).  It is easier if they are separate PRs.

Reply:
I'll take these changes out and make a new PR for them.